.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: node_js
+
+node_js:
+  lts/*
+
+before_install:
+  - npm install -g elm@0.19.0 elm-test@0.19.0 elm-format@0.8 elm-analyse@0.16
```

Comment:
Can we make the versions here same as the versions that were in `package.json`? (Or most recent)

Also, `elm-format` and `elm-analyse` aren't used here nor in the Makefile - can we remove them from here and from README? (I only had them here because my personal habit is to use tools from `node_modules/.bin/`)

Reply:
> Can we make the versions here same as the versions that were in package.json? (Or most recent)

Yes, I was attempting to install latest `0.19.0*` version but did not get it quite right.

> Also, elm-format and elm-analyse aren't used here nor in the Makefile - can we remove them from here and from README? (I only had them here because my personal habit is to use tools from node_modules/.bin/)

Yep :)